src/amo/store.js

```diff
@@ -51,6 +52,7 @@ export default function createStore({
       landing,
       languageTools,
       redirectTo,
+      recommendations,
```

Comment:
In addition, I noticed you have added the reducer here. Do you want to also add the saga in the [`rootSaga`](https://github.com/mozilla/addons-frontend/blob/2111ff06c6a9499d39a7d1db29d828285b6986ee/src/amo/sagas/index.js#L23)?